src/index.js

```diff
@@ -0,0 +1,135 @@
+
+let temperature = 20;
+let isCelsius = true;
+
+const increaseButton = document.getElementById('increaseTempControl');
+const decreaseButton = document.getElementById('decreaseTempControl');
+const cityNameInput = document.getElementById('cityNameInput');
+const headerCityName = document.getElementById('headerCityName');
+const tempValue =  document.getElementById('tempValue');
+const landscape = document.getElementById('landscape');
+const skySelect = document.getElementById('skySelect');
+const sky = document.getElementById('sky');
+const currentTempButton = document.getElementById('currentTempButton');
+
+
+const updateTemprature = () => {
```

Comment:
This method does a couple of things - it updates the temperature and also changes the display, I'd prefer a more descriptive function name to capture this.
```suggestion
const updateTemperatureAndDisplay = () => {
```